package.json

```diff
@@ -143,13 +144,73 @@
                 "category": "Azure Resource Manager Tools",
                 "title": "Reload Cached Schemas",
                 "command": "azurerm-vscode-tools.reloadSchemas"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort template",
+                "command": "azurerm-vscode-tools.sortTemplate"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort functions",
+                "command": "azurerm-vscode-tools.sortFunctions"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort outputs",
+                "command": "azurerm-vscode-tools.sortOutputs"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort parameters",
+                "command": "azurerm-vscode-tools.sortParameters"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort resources",
+                "command": "azurerm-vscode-tools.sortResources"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort variables",
+                "command": "azurerm-vscode-tools.sortVariables"
             }
         ],
         "menus": {
             "editor/context": [
                 {
                     "command": "azurerm-vscode-tools.reloadSchemas",
                     "when": "editorLangId==arm-template"
+                },
+                {
+                    "command": "azurerm-vscode-tools.sortTemplate",
+                    "when": "editorLangId==arm-template"
```

Comment:
Could you add the following to both reloadSchemas and sortTemplate in editor/context?  It would put them into the same group in the menus:

"group": "arm-template"

Reply:
Fixed.